tests/integration/targets/wafv2_rule_group/tasks/description.yml

```diff
@@ -0,0 +1,131 @@
+- name: Tests relating to setting the descriptions of wavf2_ip_sets
+  vars:
+    description_one: 'a Description - {{ resource_prefix }}'
+    description_two: 'Another_Description - {{ resource_prefix }}'
+  # Mandatory settings
+  module_defaults:
+    community.aws.wafv2_rule_group:
+      name: '{{ rule_group_name }}'
+      state: present
+      scope: REGIONAL
+      purge_rules: no
+      rules: []
+      capacity: 500
+    community.aws.wafv2_rule_group_info:
+      name: '{{ rule_group_name }}'
+      scope: REGIONAL
+  block:
+
+  - name: test setting description wafv2_rule_group (check mode)
+    wafv2_rule_group:
```

Comment:
it looks like the info module breaks when there arent any rule groups to return (throws an exception rather than returning an empty list)

Reply:
Can we do that in a follow up PR?

Reply:
yes!